jumpscale/sals/process/__init__.py

```diff
@@ -187,37 +187,33 @@ def kill(proc, sig=signal.SIGTERM, timeout=5, sure_kill=False):
         # If PID no longer exists return None immediately
         # If timeout exceeded and the process is still alive raise TimeoutExpired exception
         proc.wait(timeout=timeout)
-        j.logger.debug(f"the process with PID: {proc.pid} was terminated with sig: {sig}.")
+        j.logger.info(f"the process with PID: {proc.pid} was terminated with sig: {sig}.")
     except psutil.TimeoutExpired as e:
         # timeout expires and process is still alive.
         if sure_kill and sig != signal.SIGKILL and os.name != "nt":
             # SIGKILL not supported in windows
             # If a process gets this signal it must quit immediately and will not perform any clean-up operations
             proc.kill()
-            j.logger.debug("SIGKILL signal sent.")
+            # SIGKILL signal sent
             try:
                 proc.wait(1)
-                j.logger.debug(f"the process with PID: {proc.pid} was terminated with sig: {signal.SIGKILL}.")
+                j.logger.info(f"the process with PID: {proc.pid} was terminated with sig: {signal.SIGKILL}.")
```

Comment:
can we remove this or keep it a debug log?

Reply:
removed